pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -212,7 +212,7 @@ func (c *Cluster) Start() error {
 				namespace = c.Namespace
 			}
 			if err := c.deployPrometheusRule(prometheusRuleName, namespace); err != nil {
-				logger.Errorf("failed to deploy prometheusule. %+v", err)
+				logger.Errorf("failed to deploy prometheusRule. %+v", err)
```

Comment:
how about: "failed to deploy prometheus rule. %+v"

Reply:
good point, I will fix this.